front/src/common/components/screen-reader-snackbar/screen-reader-snackbar.hook.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ScreenReaderSnackbarOptions } from './screeen-reader-snackbar.vm';
+import { ScreenReaderSnackbarContext } from './screen-reader-snackbar.context';
+
+export const useScreenReaderSnackbarContext = () => {
+  const { options, setOptions } = React.useContext(ScreenReaderSnackbarContext);
+
+  const optionsRef = React.useRef<ScreenReaderSnackbarOptions>({
+    messages: [],
+  });
+
+  const [intervalScreen, setIntervalScreen] = React.useState(null);
+
+  return {
+    showScreeanReaderMessage: (message: string) => {
+      optionsRef.current = {
+        messages: [...optionsRef.current.messages, message],
+      };
+      setOptions(optionsRef.current);
+      const interval = setInterval(() => {
+        //Delete message
+        if (
+          optionsRef.current.messages != undefined &&
```

Comment:
I think is not necessary protected here of undefined when you are using `[...optionsRef.current.messages` before